packages/devtools/test/integration_tests/run_all.dart

```diff
@@ -3,33 +3,14 @@
 // found in the LICENSE file.
 
 @TestOn('vm')
-import 'dart:io';
-
 import 'package:test/test.dart';
 
-import 'app.dart';
-import 'debugger.dart';
-import 'integration.dart';
-import 'logging.dart';
+import 'app_test.dart' as app;
+import 'debugger_test.dart' as debugger;
+import 'logging_test.dart' as logging;
 
 void main() {
-  group('integration', () {
-    setUpAll(() async {
-      final bool testInReleaseMode =
-          Platform.environment['WEBDEV_RELEASE'] == 'true';
-
-      webdevFixture =
-          await WebdevFixture.create(release: testInReleaseMode, verbose: true);
-      browserManager = await BrowserManager.create();
-    });
-
-    tearDownAll(() async {
-      await browserManager?.teardown();
-      await webdevFixture?.teardown();
-    });
-
-    group('app', appTests);
-    group('logging', loggingTests);
-    group('debugging', debuggingTests);
-  }, timeout: const Timeout.factor(2));
+  app.main();
```

Comment:
I think it would be cleaner to leave this file as is but add main methods to the individual files that allow executing just those tests as needed.
I suspect this may lead to surprising behavior if the setUpAll and tearDownAll calls for the different tests ever diverged.

Reply:
Do you mean put the tests in the other files back into their own functions, then call those functions both from here and from the main's inside there?

Reply:
Ok the ignore whitespace diff is great! I'd just been manually looking at the commits after the commit to move the files.